packages/st2/component.makefile

```diff
@@ -88,7 +88,8 @@ bdist_wheel: .stamp-bdist_wheel
 	cat requirements.txt
 # We need to install these python packages to handle rpmbuild 4.14 in EL8
 ifeq ($(EL_VERSION),8)
-	$(PIP_BINARY) install wheel setuptools virtualenv
+        # Setuptools 60.x requires > python 3.6
+	$(PIP_BINARY) install wheel 'setuptools<60' virtualenv
```

Comment:
As the builds are green, I think we don't need the pinning anymore.
Especially, if we're planning to switch EL8 to py3.8 in the upcoming PRs.